core/src/solution_submission.rs

```diff
@@ -185,6 +185,10 @@ async fn retry_with_gas_price_increase(
     const INCREASE_FACTOR: u32 = 2;
     const BLOCK_TIMEOUT: usize = 2;
     const DEFAULT_GAS_PRICE: u64 = 15_000_000_000;
+    // openethereum requires that the gas price of the resubmitted transaction has increased by at
+    // least 12.5%. Rounded to 13% here in case of floating point error or in case the bound becomes
+    // exclusive.
+    const MIN_GAS_PRICE_INCREASE_FACTOR: f64 = 1.13;
```

Comment:
Why is the rounding buffer so big? Why not just:
```suggestion
    const MIN_GAS_PRICE_INCREASE_FACTOR: f64 = 1.125000001;
```

Or even:
```suggestion
    const MIN_GAS_PRICE_INCREASE_FACTOR: f64 = 1.125 * (1.0 + f64::EPSILON);
```

Reply:
Why is it so big? The exact value here doesn't really matter (1.125 vs 1.3) and I found this more obvious.
On the other hand I don't see anything wrong with the epsilon and I realize now there should probably also be a `ceil` in there so I'll change it.

Reply:
Changed it to calculate the effective bound once at the start instead of at every iteration and to use epsilon.

Reply:
> The exact value here doesn't really matter (1.125 vs 1.3)

I guess its not super important, it might just cause 1 fewer submission to happen because of the gas price cap (if the next gas price is `previous_gas_price * 1.125 < next_gas_price < previous_gas_price * 1.13`.